modules/subscriptions/views.php

```diff
@@ -180,6 +180,23 @@ static function render_widget_status_messages( $instance ) {
 						?>
 					</p>
 					<?php break;
+				case 'pending':
+					?>
+					<p class="error">
+						<?php
+						printf(
+							wp_kses(
+								/* translators: 1: Link to Subscription Management page https://subscribe.wordpress.com/, 2: Description of this link */
+								__( 'You subscribed this site before but you have not clicked the confirmation link yet. Please check your mailbox. <br /> Otherwise, you can manage your preferences at <a href="%1$s" title="%2$s" target="_blank">subscribe.wordpress.com</a>.', 'jetpack' ),
```

Comment:
Let's add a `rel` attribute here as well.

```suggestion
								__( 'You subscribed this site before but you have not clicked the confirmation link yet. Please check your mailbox. <br /> Otherwise, you can manage your preferences at <a href="%1$s" title="%2$s" target="_blank" rel="noopener noreferrer">subscribe.wordpress.com</a>.', 'jetpack' ),
```

Reply:
Oopss.... thanks! I missed this again.